backend/groth16/bn254/mpcsetup/unit_test.go

```diff
@@ -0,0 +1,286 @@
+package mpcsetup
```

Comment:
File only generated for BN254? I would code-generate just in case.

Reply:
The idea was not to clog the CI with unit tests. The full test is still generated for all curves. Ignoring the unit tests saves 5 seconds per curve on my machine. Perhaps not worth it?

Reply:
In that case, I would still code generate, but maybe conditionally add `test.Skip()` in case the curve isn't BN254. Then atleast we ensure that everything is syntactically correct and the unit test is covered in code generation.